src/clientManager.js

```diff
@@ -34,11 +34,33 @@ ClientManager.prototype.init = function (identHandler, sockets) {
 };
 
 ClientManager.prototype.findClient = function (name) {
-	return this.clients.find((u) => u.name === name);
+	name = name.toLowerCase();
+	return this.clients.find((u) => {
+		const clientName = u.name;
+		return clientName.toLowerCase() === name;
+	});
 };
 
 ClientManager.prototype.loadUsers = function () {
-	const users = this.getUsers();
+	const usersList = this.getUsers();
+	const updatedUsersList = [];
+
+	for (let i = 0; i < usersList.length; i++) {
+		const index = [];
+
+		for (let j = 0; j < usersList.length; j++) {
```

Comment:
Double loop may get more expensive than it needs to be.

You can use a `Set` to avoid the inner loop entirely. Like so:

Lowercase the name and check whether its in that set, if it is, then remove it from the user list and print a message, otherwise add it to set and continue.

If you loop from the end, then you can splice out the user from the array while iterating.

Reply:
hi @xPaw ,
I can rewrite it using a single loop and the end-user array list will be in lowercase. Is that fine?